app/template/rest.py

```diff
@@ -205,8 +205,9 @@ def preview_letter_template_by_notification_id(service_id, notification_id, file
 
             pdf_file = get_letter_pdf(notification)
 
-        except botocore.exceptions.ClientError:
-            current_app.logger.info
+        except botocore.exceptions.ClientError as e:
+            current_app.logger.exception(
+                'Error getting letter file from S3 notification id {}'.format(notification_id), e)
```

Comment:
`exception` log doesn't need the exception - it pulls it automatically out of `sys.exc_info`. So we just want `current_app.logger.exception('My error msg {}'.format(my_args))`